cms/templates/js/unit-access-editor.underscore

```diff
@@ -0,0 +1,133 @@
+<%
+var groupAccess = xblockInfo.get('group_access');
+var selectablePartitions = [];
+var groupIdToName = {};
+
+contentGroups.forEach(function (partition) {
+    if (partition['scheme'] != 'cohort' && partition['scheme'] != 'enrollment_track') {
+        return;
+    }
+    if (partition['scheme'] == 'enrollment_track') {
+        var tracks = [];
+        var trackSelected = false;
+        partition['groups'].forEach(function (group) {
+            if (group['selected']) {
+                trackSelected = true;
+            }
+        });
+        // Only add enrollment tracks if there are multiple tracks to restrict from
+        if (partition['groups'].length < 2 && !trackSelected) {
+            return;
+        }
+    }
+    selectablePartitions.push(partition);
+});
+
+selectablePartitions.forEach(function (partition) {
+    var schemeId = partition['id'];
+    var groups = partition['groups'];
+    groups.forEach(function (group) {
+        var groupId = group['id'];
+        var groupName = group['name'];
+        var fullId = schemeId.toString() + '-' + groupId.toString();
```

Comment:
While I am not in love with this concatenation (I find map/dictionary structures much more natural for this sort of hierarchical data), at least you have a dividing character this time.